src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.csproj

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```

Comment:
Why are you removing these defaults? What happens now if you build one of these csproj's without passing in a configuration or osgroup?